setup.py

```diff
@@ -58,7 +58,15 @@
         "datasets>=1.2.1",
         "evaluate",
         "protobuf>=3.20.1",
-        "accelerate",  # ORTTrainer requires it.
+        "transformers>=4.36,<4.49.0",
+    ],
+    "onnxruntime-training": [
```

Comment:
The readme and docs should reflect this change

Reply:
added in https://github.com/huggingface/optimum/pull/2173